packages/svelte2tsx/repl/debug.ts

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs';
+import svelte2tsx from '../src';
+const content = fs.readFileSync(`${__dirname}/index.svelte`, 'utf-8');
+svelte2tsx(content);
+/**
+ * To enable the REPL, simply run the "dev" package script.
+ *
+ * The "/repl/index.svelte" file will be converted to tsx
```

Comment:
This comment needs to be updated since you now use the vs code debug functionality

Reply:
The REPL still updates through rollup, the debug thing is secondary